lib/interface/cli/helpers/general.js

```diff
@@ -106,6 +106,27 @@ const prepareKeyValueObjectsFromCLIEnvOption = (environmentVariables) => {
     return variables;
 };
 
+const prepareKeyValueObjectsFromEnvFileOption = (environmentVariables) => _.map(environmentVariables, (value, key) => {
+    if (_.isObject(value)) {
+        const { val, encrypted } = value;
+        return { key, value: val, encrypted };
+    }
+    return { key, value };
+});
+
+const prepareEncryptedValues = (requestedVariables, encrypted) => {
```

Comment:
function name a bit too generic - let's change it to something more self explanatory